tests/userSIgnUpTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+use App\user;
+use Laravel\Socialite\Facades\Socialite;
+use Laravel\Lumen\Testing\DatabaseMigrations;
+use Laravel\Lumen\Testing\DatabaseTransactions;
+
+class UserSignUpTest extends TestCase
+{
+    use DatabaseTransactions;
+    use DatabaseMigrations;
+
+    /**
+     * Test that user sign up successfully
+     */
+    public function testForEmptyFields()
+    {
+        $user = factory('App\User')->create();
+
+        $response = $this->call('POST', '/api/v1/register', [
+            
+        ]);
+
+        $json = json_decode($response->getContent());
```

Comment:
What's this used for? I'm guessing it's either `$this->response->getContent()` or `$json`, right?